WidgetKit/SearchQuickLinksMedium/SearchQuickLinks.swift

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import WidgetKit
+import SwiftUI
+import Shared
+
+struct Provider: TimelineProvider {
+    public typealias Entry = SimpleEntry
+
+    public func snapshot(with context: Context, completion: @escaping (SimpleEntry) -> ()) {
+        let entry = SimpleEntry(date: Date())
+        completion(entry)
+    }
+
+    public func timeline(with context: Context, completion: @escaping (Timeline<Entry>) -> ()) {
+        let entries = [SimpleEntry(date: Date())]
+        let timeline = Timeline(entries: entries, policy: .atEnd)
+        completion(timeline)
+    }
+}
+
+struct SimpleEntry: TimelineEntry {
+    public let date: Date
+}
+
+struct SearchQuickLinksEntryView : View {
+    @ViewBuilder
+    var body: some View {
+        VStack {
+            HStack(alignment: .top, spacing: 8.0) {
+                ImageButtonWithLabel(isSmall: false, link: .search)
+                ImageButtonWithLabel(isSmall: false, link: .privateSearch)
+            }
+            HStack(alignment: .top, spacing: 8.0) {
+                ImageButtonWithLabel(isSmall: false, link: .copiedLink)
+                ImageButtonWithLabel(isSmall: false, link: .closePrivateTabs)
+            }
+        }
+        .padding(10.0)
+        .background(Color("backgroundColor"))
+    }
+}
+
+struct SearchQuickLinksWigdet: Widget {
+    private let kind: String = "Quick Actions - Medium"
+
+    public var body: some WidgetConfiguration {
+        StaticConfiguration(kind: kind, provider: Provider(), placeholder: SearchQuickLinksEntryView()) { entry in
+            SearchQuickLinksEntryView()
+        }
+        .supportedFamilies([.systemMedium])
+        .configurationDisplayName("Firefox - Quick Actions")
+        .description("Access to frequent actions directly on your home screen.")
```

Comment:
@noorhashem Will this get replaced with what we have in strings file?

Reply:
@nishant2718  yes